.github/workflows/close.yaml

```diff
@@ -0,0 +1,51 @@
+name: "Close"
+
+on: # yamllint disable-line rule:truthy
+  pull_request:
+    types:
+      - "opened"
+      - "reopened"
+
+jobs:
+  close:
+    name: "Close"
+
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - name: "Close pull request based on actors"
+        uses: "actions/github-script@v2.0.1"
+        with:
+          github-token: "${{ secrets.ERGEBNIS_BOT_TOKEN }}"
+          script: |
+            const actors = [
+              "localheinz",
```

Comment:
This should probably be replaced with names of other users, especially those that you

- do not want to deal with
- do not want to block entirely in a repository running this workflow